srcpkgs/ibus/template

```diff
@@ -1,12 +1,13 @@
 # Template file for 'ibus'
 pkgname=ibus
-version=1.5.23
-revision=4
+version=1.5.26
+revision=1
 build_style=gnu-configure
 build_helper="gir"
-configure_args="--enable-ui --enable-gtk3 --disable-tests
- --disable-schemas-compile --enable-memconf --enable-dconf
- --enable-wayland --with-python=/usr/bin/python3
+configure_args="--enable-ui --enable-gtk3
+ --disable-tests --disable-schemas-compile --disable-systemd-services
```

Comment:
let's keep the systemd services, so they can be used as a reference for creating services for ibus

Reply:
Alright, thanks.

Reply:
If this is enabled then `systemd.pc` is required. (which isn't provided by void)
I don't see a way to work around that without patches:
https://github.com/ibus/ibus/blob/master/configure.ac#L396-L410

Reply:
ok, then leave it as-is